onnxruntime/test/providers/cpu/tensor/resize_op_test.cc

```diff
@@ -418,7 +418,7 @@ TEST(ResizeOpTest, ResizeOpNearestDownSampleTest_WithSizes) {
 }
 
 TEST(ResizeOpTest, ResizeOpNearestDownSampleTest_tf_half_pixel) {
-  OpTester test("Resize", 11);
```

Comment:
this needs to remain opset-12 because it uses an attribute deprecated in opset-13